### DISCUSSION TITLE
Welcome to Nylo Discussions!

### DISCUSSION THREAD

**Discussion by agordn52** [nylo-core/nylo#16] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by parcox** [comment, marked as answer]
Hi @agordn52 ,

My name is Bowo, I'm from Indonesia. I'm new to Flutter. Thanks for a this cool project. There are two main issues for me, and perhaps for the others too,

1. It's lack of examples project. If there are any project that's build using Nylo, I think it'll be great if we have something like a showcase to help a new comer to learn faster by customizing existing project. Or maybe, you could make a video series on how to build a simple usable Flutter app by using Nylo from a scracth, and share it publicly on  Youtube or other platform.
2. The documentation needs to be more up-to-date. It seems that you already release Nylo v.3 but at the documentation website we still could not find the latest release documentation even there are some new features introduced in the latest version.

I'm hoping to see Nylo growing to get bigger and widely known project.

**Reply by agordn52** [reply]
Hi @parcox,

Thanks for your comment Bowo, you're right.

1. I think a YouTube series would be really cool. I'll plan something for next week but if anyone else fancied making videos too that'd be awesome. In terms of example projects, yeah there are a few out in the wild, I will link them here when I get the chance.
2. I'm currently finishing the new docs for v3.0.0, it's taking me a little longer than usual because I've decided to redo the whole look of the documentation page. When this is complete, I'll ping a notification on here to let people know it's done. I swear writing docs is harder than coding haha. I'll let you know when it's live 👍

Lastly, thanks. If you have any ideas feel free to drop a comment at https://github.com/nylo-core/nylo/discussions/categories/ideas or  send a PR on any of the repos

**Reply by agordn52** [reply]
@parcox 

Docs are live for v3.x
https://nylo.dev/

**Comment by bhargavraviya** [comment]
Hello @agordn52 and everyone,

My name is Bhargav Raviya. From India, a Laravel and Flutter developer

 

Their given basic options make basic staff and features like firebase, share_plus, url launcher, push notification, connection check, etc. necessary to every app project.

 

If you need help developing this package, tell me. Because I am in the learning stage

**Reply by agordn52** [reply]
Hi @bhargavraviya,

Nice to meet you, glad to hear you're interested in contributing.
I'll check through some of the PRs you've sent at my earliest convenience.

Next week I'll put together a roadmap for v6, planning to launch that version in April/May 2024.

If anyone has any pain points with Nylo, feel free to drop your ideas 💡
